dab/voice.py

```diff
@@ -0,0 +1,29 @@
+from schema import dab_response_validator
+from time import sleep
+from DabTester import YesNoQuestion, Default_Validations
+
+def send_audio(test_result, durationInMs=0,expectedLatencyMs=0):
+    dab_response_validator.validate_dab_response_schema(test_result.response)
+    sleep(5)
+    if type(expectedLatencyMs) == int:
+        return YesNoQuestion(test_result, f"Can you verify the voice command has been initated?") and Default_Validations(test_result, durationInMs, expectedLatencyMs)
+    else:
+        return YesNoQuestion(test_result, expectedLatencyMs)
+
+def send_text(test_result, durationInMs=0,expectedLatencyMs=0):
+    dab_response_validator.validate_dab_response_schema(test_result.response)
+    sleep(5)
+    if type(expectedLatencyMs) == int:
+        return YesNoQuestion(test_result, f"Can you verify the voice command ${test_result.request} has been initated?") and Default_Validations(test_result, durationInMs, expectedLatencyMs)
+    else:
+        return YesNoQuestion(test_result, expectedLatencyMs)
+
+def list(test_result, durationInMs=0,expectedLatencyMs=0):
+    dab_response_validator.validate_list_voice_response_schema(test_result.response)
+    sleep(5)
+    return Default_Validations(test_result, durationInMs, expectedLatencyMs)
```

Comment:
can dab_response_validator be moved inside Default_Validations? Does this need to be done separately?

Reply:
I mean, it kind of more unique to the operations themselves. Actually, I want to ask you here, when the format is not correct, should we stop the whole test or continue?

Reply:
Just looked at schema.py. github had hidden it. The changes make sense. My opinion would be to provide a command line option for immediate fail vs continue.